manual/src/cmds/runtime-tracing.etex

```diff
@@ -70,6 +74,31 @@ The full set of events emitted by probes and their documentation can be found in
  section~\ref{Runtime_events}.
 \fi
 
+\item[Custom probes] are defined using the
+\texttt{Runtime_events.User} module.
+Identified by a globally unique name, they record events and provide payloads of
+built-in (\texttt{unit}, \texttt{int}, \texttt{span}) and user-defined types.
+Consumers and producers are not compiled together, so they don't usually have
+the same set of registered events. The event name is used by the consumer to
+identify producer's events, with a fallback mechanism for unknown events of
+built-in types. Unknown events for user-defined types are currently ignored,
+even if the type is defined both in the producer and consumer side.
+
+An extensible variant \texttt{tag} provides a way to attach additional data to
+events, which can be used to provide additional information about the event's
+origin. The tag is not serialized, so it is only available for \textit{known}
```

Comment:
"event's origin" is meh; I think that "event origin" works, otherwise "the origin of the event". (Why not just stop the sentence at "attach additional data to events."?)

Reply:
Yes let's just say additional data.